### ISSUE TITLE
Enable fine-tuning for the integrated scorer

### ISSUE THREAD

**Issue by LTLA** [SingleR-inc/singlepp#13]
Might as well do provide the option. This could improve robustness to nonsense references. Probably could re-use `FineTuner` and just template the marker refinement strategy given the "labels" (i.e., references) in use at each iteration.

**Comment by LTLA** [comment]
Closed by #22.
